serialize-utils/src/cursor.rs

```diff
@@ -133,7 +133,11 @@ mod test {
     #[test]
     fn test_read_option_u64() {
         for _ in 0..100 {
-            let test_value = rand::random::<Option<u64>>();
+            let test_value = if rand::random_bool(0.5) {
+                Some(rand::random::<u64>())
+            } else {
+                None
+            };
```

Comment:
```suggestion
            let test_value = rand::rng()
                .random_bool(0.5)
                .then(|| rand::random::<u64>());
```

Very minor nit: I guess this could be written slightly more idiomatically.